frontend/src/components/Sunscreen.jsx

```diff
@@ -0,0 +1,29 @@
+export default function Sunscreen({ sunscreen }) {
+  const price = sunscreen.pricePerOz.$numberDecimal
+  
+  return (
+    <div className=''>
+      <div className='max-w-md my-8 mx-auto border-[1px] border-lt-blue rounded-xl shadow-md overflow-hidden md:max-w-2xl'>
+        <div className='md:flex'>
+          <div className='md:shrink-0'>
+            <img className='h-48 w-full p-2 object-cover md:h-full md:w-48' src={sunscreen.image} alt={`Packaging of ${sunscreen.name}`}/>
+          </div>
+          <div className='p-8'>
+            <div className='uppercase tracking-wide text-sm font-semibold hover:underline'>{sunscreen.brand}</div>
+            <a href='#' className='block mt-1 text-lg/tight font-medium hover:underline'>{sunscreen.name}</a>
+            <ul>
+              <li className='mt-2'>Form: {sunscreen.form}</li>
+              <li className='mt-2'>Type: {sunscreen.type}</li>
+              <li className='mt-2'>SPF: {sunscreen.spf}</li>
+              <li className='mt-2'>Price: {price < 5 ? '$' :
```

Comment:
I like this bit of conditional price checking :)

Reply:
Though I do wonder if there's a more concise way to generate the number of "$" signs... will have to think!